tools/marvin/marvin/config/BareMetal.cfg

```diff
@@ -0,0 +1,194 @@
+                            "hypervisor": "BareMetal",
+                            "hosts": [
+                                {
+                                    "username": "root",
+                                    "url": "http://10.147.42.23",
+                                    "password": "calvin",
+                                    "hostmac": "D4:AE:52:BC:E5:52",
+                                    "cpunumber": "1",
+                                    "cpuspeed": "1000",
+                                    "memory": "1024"
+                                }
+                            ],
+                            "clustertype": "CloudManaged"
+                        }
+                    ],
+                    "gateway": "10.147.42.1"
+                }
+            ],
+            "internaldns1": "10.140.50.6"
+        }
+    ],
+    "dbSvr": {
+        "dbSvr": "10.147.38.153",
+        "passwd": "password",
+        "db": "cloud",
+        "port": 3306,
+        "user": "cloud"
+    },
+    "logger": {
+        "LogFolderPath": "/tmp/"
+    },
+    "globalConfig": [
+        {
+            "name": "storage.cleanup.interval",
+            "value": "300"
+        },
+        {
+            "name": "vm.op.wait.interval",
+            "value": "5"
+        },
+        {
+            "name": "default.page.size",
+            "value": "10000"
+        },
+        {
+            "name": "instance.name",
+            "value": "QA"
+        },
+        {
+            "name": "workers",
+            "value": "10"
+        },
+        {
+            "name": "account.cleanup.interval",
+            "value": "600"
+        },
+        {
+            "name": "guest.domain.suffix",
+            "value": "sandbox.kvm"
+        },
+        {
+            "name": "expunge.delay",
+            "value": "60"
+        },
+        {
+            "name": "vm.allocation.algorithm",
+            "value": "random"
+        },
+        {
+            "name": "expunge.interval",
+            "value": "60"
+        },
+        {
+            "name": "project.invite.timeout",
+            "value": "300"
+        },
+        {
+            "name": "network.gc.interval",
+            "value": "60"
+        },
+        {
+            "name": "network.gc.wait",
+            "value": "60"
+        },
+        {
+            "name": "expunge.workers",
+            "value": "3"
+        },
+        {
+            "name": "check.pod.cidrs",
+            "value": "true"
+        },
+        {
+            "name": "secstorage.allowed.internal.sites",
+            "value": "10.147.28.7/32"
+        },
+        {
+            "name": "direct.agent.load.size",
+            "value": "1000"
+        },
+        {
+            "name": "enable.dynamic.scale.vm",
+            "value": "true"
+        },
+        {
+            "name": "execute.in.sequence.hypervisor.commands",
+            "value": "true"
+        }
```

Comment:
@sanju1010 , we also need to set the configuration parameter "baremetal.provision.done.notification.enabled" to false. This parameter is applicable only for Baremetal advanced zone.